crates/aptos/src/config/mod.rs

```diff
@@ -174,6 +184,9 @@ pub struct GlobalConfig {
     /// Whether to be using Global or Workspace mode
     #[serde(skip_serializing_if = "Option::is_none")]
     pub config_type: Option<ConfigType>,
+    /// Prompt response type
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub default_prompt_response: Option<PromptResponseType>,
```

Comment:
Can we use `serde(default)` here instead of making this an `Option`? As it is now the `Option` doesn't really make much sense given there is also a `Prompt` variant of the `PromptReponseType` enum.

Reply:
In that case, it would save `default_prompt_response: Prompt` in the config file, whenever we run `set-global-config`. It's what we should come up with?